### PR TITLE
Remove PHP 5 from static keyword page

### PR DESCRIPTION
**Pull request by tiffany-taylor**
I'll start working on removal of PHP 5 from OOP section, this was a file I encountered while browsing the manual. I'll add more files as I go through.

### CODE REVIEW

**Comment by tiffany-taylor** [comment]
Changing this to just the static keyword page as I probably won't have the time to go over all of the OOP5 pages.

File: language/oop5/static.xml [hunk 1]
```diff
@@ -84,14 +77,13 @@ $classname::aStaticMethod(); // As of PHP 5.3.0
 
    <para>
     Like any other PHP static variable, static properties may only be
-    initialized using a literal or constant before PHP 5.6; expressions are
-    not allowed. In PHP 5.6 and later, the same rules apply as &const;
-    expressions: some limited expressions are possible, provided they can be
-    evaluated at compile time.
+    initialized using a literal or constant; expressions are not allowed. 
+    The same rules apply as &const; expressions: some limited expressions 
+    are possible, provided they can be evaluated at compile time.
```
Comment (cmb69) [inline language/oop5/static.xml:82]: That doesn't look quite right. The initialization restriction to "literal or constant" only applies prior to PHP 5.6.

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thanks, very nice, except for the commented issue.

File: language/oop5/static.xml (hunk 1 above)
Comment (tiffany-taylor) [reply language/oop5/static.xml:82]: Does this mean static properties can be initialized with an expression as well? Trying to see how I should wordsmith it.

Comment (cmb69) [reply language/oop5/static.xml:82]: With a constant expression, i.e. "some limited expressions are possible, provided they can be evaluated at compile time"

Comment (tiffany-taylor) [reply language/oop5/static.xml:82]: > Static variables have limited expression support, provided they can be evaluated at compile time.
How is that?

Comment (tiffany-taylor) [reply language/oop5/static.xml:82]: Or maybe:
> Static variables have limited support for expressions, provided they can be evaluated at compile time.

Comment (cmb69) [reply language/oop5/static.xml:82]: Well, actually I think we should define "constant expression" in a central place, and then just link to it. Currently, the most fitting definition (but IMO still a bit unspecific) is in https://www.php.net/manual/en/language.oop5.constants.php, but probably should be in https://www.php.net/manual/en/language.constants.php, but that page doesn't even mention global `const` definitions, which are explained in https://www.php.net/manual/en/language.constants.syntax.php. Total mess :(

Comment (afilina) [reply language/oop5/static.xml:82]: Pinging @tiffany-taylor in case this fell through the cracks.

**Comment by Girgias** [comment]
Closing as this should have been covered by #277

**Pull request closed without merging**